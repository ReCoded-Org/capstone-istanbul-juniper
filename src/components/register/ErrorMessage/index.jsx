import React from "react";

const ErrorMessage = ({ message }) => {
  if (message) {
    return (
      <div className="loginContainer__loginDialog__errorContainer">
        {errorMessage}
      </div>
    );
  } else return null;
};
