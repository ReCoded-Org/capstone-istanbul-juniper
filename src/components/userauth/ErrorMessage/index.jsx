import React from "react";

const ErrorMessage = ({ message }) => {
  if (!message) {
    return null;
  }
  return (
    <div className="loginContainer__loginDialog__errorContainer">{message}</div>
  );
};

export default ErrorMessage;
