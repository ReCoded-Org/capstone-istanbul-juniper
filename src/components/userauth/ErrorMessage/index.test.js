import React from "react";
import renderer from "react-test-renderer";
import ErrorMessage from ".";

it("ErrorMessage snapshot test", () => {
  const tree = renderer.create(<ErrorMessage message={"test"} />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("ErrorMessage without message snapshot test ", () => {
  const tree = renderer.create(<ErrorMessage />).toJSON();
  expect(tree).toMatchSnapshot();
});
