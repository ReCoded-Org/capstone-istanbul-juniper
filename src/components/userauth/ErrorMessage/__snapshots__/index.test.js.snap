// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage snapshot test 1`] = `
<div
  className="loginContainer__loginDialog__errorContainer"
>
  test
</div>
`;

exports[`ErrorMessage without message snapshot test  1`] = `null`;
